tests/testthat/test-help-handlers.R

```diff
@@ -69,3 +69,27 @@ Section1:
   expect_match(result$returns, "array-like values")
 })
 
+test_that("Parameter help can be extracted from Python objects", {
+  skip_if_no_docutils()
+  
+  py_run_string('
+def foo(alpha, omega):
+  """
+Parameters
+----------
+
+alpha: int, optional
+  The first argument.
+
+omega: int, optional
+  The last argument.
+"""
+  pass
```

Comment:
This might be a bit nitpick but could you indent the docs to match with the indentation level of `pass`?